low_level/comms.py

```diff
@@ -24,14 +24,47 @@
 import serial
 import config
 from low_level.frameformat import PROTOCOL_DELIMITER, MAX_DATA_TYPES, DataType, DataTypeSize
+from config import OS, RaspberryPi
+try:
```

Comment:
I would add commentary here to explain what is happening.

Reply:
Sounds good. I guess you should make the change and then I will close this comment.

Reply:
Comment added.